lib/bookingsync/engine/models/base_account.rb

```diff
@@ -10,7 +10,7 @@ def token
       token_options = {}
       if oauth_refresh_token
         token_options[:refresh_token] = oauth_refresh_token
-        token_options[:expires_at]    = oauth_expires_at
+        token_options[:expires_at]    = oauth_expires_at && oauth_expires_at.to_i
```

Comment:
would be great to have a spec for that, it's a really surprising change in oauth

Reply:
@Azdaroth We do have a spec for that (and it was failing, otherwise I wouldn't notice).
The current code mirrors what was used in previous versions of the oauth2 gem (pre 1.4.4/2).

Reply:
oh, ok, so the spec is there, cool 👍